notebooks/College_Demo.ipynb

```diff
@@ -21,7 +21,10 @@
     "\n",
```

Comment:
Can we add a description at the top motivating the use of SQL as an execution backend and why its important? Similar to [this](https://lux-api.readthedocs.io/en/latest/source/advanced/executor.html).